docs/docs-content/vertex/install-palette-vertex/install-on-kubernetes/uninstall.md

```diff
@@ -0,0 +1,122 @@
+---
+sidebar_label: "Uninstallation"
+title: "Uninstall VerteX"
+description: "Learn how to uninstall a VerteX installation from your cluster using Helm charts."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 40
+tags: ["self-hosted", "enterprise"]
+keywords: ["self-hosted", "enterprise"]
+---
+
+To uninstall VerteX from your cluster, you need to uninstall VerteX management plane and Cert Manager. Optionally, you
+may have installed Image Swap and Reach system, which also need to be uninstalled. This uninstall process applies to
+both connected and airgap instances of self-hosted VerteX.
+
+:::warning
+
+This installation process only applies to VerteX instances installed using Helm charts. If you used the Palette CLI to
+install VerteX, this process does not apply.
+
+:::
+
+## Prerequisite
+
+- An active self-hosted VerteX instance.
+
+- Access to the Kubernetes cluster where VerteX is deployed.
+
+- Permissions to perform delete actions on all relevant cluster resources.
+
+- Helm is installed and available on your host machine.
+
+- kubectl is installed and available on your host machine.
+
+## Uninstall VerteX
+
+1. Ensure you are using the appropriate context where VerteX is deployed.
+
+   ```shell
+   kubectl config current-context
+   ```
+
+2. Issue the following command to start uninstalling the Vertex management plane. Similar to Reach, this will only
```

Comment:
```suggestion
2. Issue the following command to start uninstalling the Vertex management plane. This will only
```
I would remove this as Reach is mentioned after :)